src/OrchardCore.Modules/OrchardCore.Alias/Settings/AliasPartSettingsDisplayDriver.cs

```diff
@@ -46,15 +47,15 @@ public override async Task<IDisplayResult> UpdateAsync(ContentTypePartDefinition
 
             var model = new AliasPartSettingsViewModel();
 
-            if (await context.Updater.TryUpdateModelAsync(model, Prefix, m => m.Pattern))
+            if (await context.Updater.TryUpdateModelAsync(model, Prefix, m => m.Pattern, m=>m.Options))
             {
```

Comment:
formatting here `, m => m.Options`

Reply:
Thank you @deanmarcussen ... I have to remove my subscription to the ugly code society 😅